### ISSUE TITLE
No Python modules to install

### ISSUE THREAD

**Issue by hdlynxx** [bhavintandel/py-packager#1]
When I run the command: python setup.py bdist_egg I get the following error. Any suggestions?

C:\Program Files\Python39\lib\distutils\dist.py:274: UserWarning: Unknown distribution option: 'include_data_files'
  warnings.warn(msg)
running bdist_egg
running egg_info
writing awswrangler_geopandas.egg-info\PKG-INFO
writing dependency_links to awswrangler_geopandas.egg-info\dependency_links.txt
writing requirements to awswrangler_geopandas.egg-info\requires.txt
writing top-level names to awswrangler_geopandas.egg-info\top_level.txt
reading manifest file 'awswrangler_geopandas.egg-info\SOURCES.txt'
writing manifest file 'awswrangler_geopandas.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
warning: install_lib: 'build\lib' does not exist -- no Python modules to install

**Comment by bhavintandel** [comment]
It must have generated `dist/dependencies-0.1.0-py3.6.egg`. 

The above warning may be avoided by:
```
pip install --upgrade pip
pip install --upgrade setuptools
```

**Comment by hdlynxx** [comment]
Doesn't seem the case, when I pip install --upgrade it states that the requirements are already satisfied

**Comment by bhavintandel** [comment]
Have updated the code and removed some unwanted keyword. Can you try with new updates ?

**Comment by hdlynxx** [comment]
Yes I've tried it but it doesn't seem to fix the issue. I run python setup.py bdist_egg from the folder py-packager in the cmc. That is okay?

**Comment by bhavintandel** [comment]
Yeah, running `python setup.py bdist_egg` from inside the folder will actually generate build under **dist/** folder

**Comment by hdlynxx** [comment]
Still no fix, does the python version matter?

**Comment by bhavintandel** [comment]
So this code is not been tested on Windows OS yet. The commands mostly were tested on Linux distribution. 
Are you getting a particular error? As above logs just have warnings.
